### PR TITLE
Fix Issue #22

### PR DESCRIPTION
**Pull request by fAns1k**
Issue #22 

Added Name and AddressZip , and also others:
* @param addressLine1 - first line of the billing address
* @param addressLine2 - second line of the billing address
* @param addressCity - city of the billing address
* @param addressState - state of the billing address
* @param addressCountry - country for the billing address
* @param brand - brand of this card
* @param last4 - last 4 digits of the card
* @param fingerprint - the card fingerprint
* @param funding - the funding type of the card
* @param country - ISO country code of the card itself
* @param currency - currency used by the card

### CODE REVIEW

**Comment by itsmepetrov** [comment]
Looks good to merge

**Comment by cybergrind** [comment]
Is this really affect only java code?
Why we cannot test this code?

File: android/src/main/java/com/gettipsi/stripe/StripeModule.java [hunk 1]
```diff
@@ -373,4 +371,36 @@ public void onResult(@NonNull BooleanResult booleanResult) {
       }
     );
   }
+
+  private String exist(final ReadableMap map, final String key) {
+    if (map.hasKey(key)) {
+      return map.getString(key);
+    } else {
+      return null;
```
Comment (cybergrind) [inline android/src/main/java/com/gettipsi/stripe/StripeModule.java]: @fAns1k I believe method that returns `null` should have explicit comment about that or return empty string.

File: ios/TPSStripe/TPSStripeManager.m [hunk 2]
```diff
@@ -92,7 +92,6 @@ - (dispatch_queue_t)methodQueue {
     [cardParams setAddressCountry: params[@"addressCountry"]];
     [cardParams setAddressZip: params[@"addressZip"]];
 
-    // cardParams.expMonth = [params[@"expMonth"] integerValue];
     // cardParams.expYear = [params[@"expYear"] integerValue];
```
Comment (itsmepetrov) [inline ios/TPSStripe/TPSStripeManager.m]: I think we can also remove this line

File: example/src/CustomCardScreen.js [hunk 3]
```diff
@@ -86,6 +107,17 @@ export default class CustomCardScreen extends Component {
             </Text>
           }
         </View>
+        <Text style={styles.instruction}>
+          {token == null ? 'Click button to get token based on params.' : ''}
+        </Text>
```
Comment (itsmepetrov) [inline example/src/CustomCardScreen.js]: Rewrite to:

```js
{!token &&
  <Text style={styles.instruction}>
    Click button to get token based on params.' : ''}
  </Text>
}
```

File: example/src/CustomCardScreen.js [hunk 4]
```diff
@@ -108,6 +140,12 @@ const styles = StyleSheet.create({
     color: '#333333',
     marginBottom: 5,
   },
+  optionalParams: {
+    fontSize: 10,
```
Comment (itsmepetrov) [inline example/src/CustomCardScreen.js]: Font size is too small, and different of mandatory fields, may be we should create common style for mandatory and optional fields?

**Comment by itsmepetrov** [comment]
I think now we can merge this PR :)

**Pull request merged**